spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -6,31 +6,44 @@
    To look at delicious donuts
   } do
   # ACCEPTANCE CRITERIA:
-  # Root is Donut#index
-  # See Links on Donut names
+  # [X] Root is Donut#index
+  # [X] Unauthenticated user is directed to the Donut#index
+  # [X] Authenticated user can add a new donut
+  # [X] See Links on Donut names
+  # [X] Clicking donut name link redirects to donut's show page
 
   let!(:donut) { create(:donut) }
 
-  scenario "sees a list of donuts and a link for a new donut" do
-    visit root_path
+  context "as an unauthenticated user" do
+    scenario "I will be directed to the root and see a list of donuts" do
```

Comment:
You're not actually being directed there, you're visiting there. Try `when visiting the root`...